exceptiongroup/_tests/test_exceptiongroup.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from exceptiongroup import ExceptionGroup
+
+
+def test_exception_group_init():
+    memberA = ValueError("A")
+    memberB = RuntimeError("B")
+    group = ExceptionGroup(
+        "many error.", [memberA, memberB], [str(memberA), str(memberB)]
+    )
+    assert group.exceptions == [memberA, memberB]
+    assert group.message == "many error."
+    assert group.sources == [str(memberA), str(memberB)]
```

Comment:
Can you also add a check here that `group.args == ...`? Otherwise looks good to me.